plugins/generator-1.19.4/forge-1.19.4/templates/projectile/projectile.java.ftl

```diff
@@ -38,6 +38,8 @@ package ${package}.entity;
 @OnlyIn(value = Dist.CLIENT, _interface = ItemSupplier.class)
 public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 
+	public static final ItemStack PROJECTILE_ITEM = ${mappedMCItemToItem(data.projectileItem)}.getDefaultInstance();

```

Comment:
Why is getDefaultInstance call needed instead of mappedMCItemToItemStackCode how it was done before and is done in other locations where itemstack is needed?

Reply:
It is needed in the item mod element (arrowshootcode macro if I remember correctly)